packages/experimental/kai/src/pages/IdentityPage.tsx

```diff
@@ -48,7 +47,7 @@ const IdentityPage = () => {
             </Button>
             <Button
               variant='primary'
-              onClick={() => navigate(createSpacePath(space?.key, defaultFrameId))}
+              onClick={() => navigate(createPath({ spaceKey: space!.key, frame: defaultFrameId }))}
```

Comment:
Can be guarantee this returns a space? If not this seems like it will break sooner or later

Reply:
i've changed this to have a handler, but we should discuss the onboarding flow -- esp. post shell integration.